assets/stylesheets/_modules.scss

```diff
@@ -0,0 +1,3 @@
+@import 'components/team';
+@import 'components/buttons';
+@import 'components/typography';
```

Comment:
whitespace return at end of all files. this is a configuration option in sublime text